app/views/layouts/application.html.erb

```diff
@@ -6,21 +6,23 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <%= render "shared/cdn_assets" %>
```

Comment:
Great job adding your CDN assets to a partial! This helps keep the layout file clean :)